tests/fuzz/wasm-mutator-fuzz/smith_wasm.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# 1.check parameter
+if [ ! $1 ]; then
+	echo "Parameter is empty, please enter parameter !"
+    exit 
+fi
+
+# 2.check dir
+buildPath="./build"
+corpusPath="$buildPath/CORPUS_DIR"
+if [[ ! -d "$buildPath" ]]; then
+    echo "auto create the build folder !"
+    mkdir build
+else # build Folder exists
+    if [[ -d "$buildPath" ]]; then # CORPUS_DIR exists
+        rm -rf $corpusPath
+    fi
+fi
+
+# 3.change dir
+# cd build && mkdir CORPUS_DIR && cd CORPUS_DIR
+cd build && mkdir CORPUS_DIR && cd CORPUS_DIR
+
+# 4.generate *.wasm file
+echo "Generate $@ files according to user requirements"
+
+for((i=1; i<($@+1); i++));
+do
+head -c 100 /dev/urandom | wasm-tools smith -o test_$i.wasm
```

Comment:
Here. `wasm-tools smith` only generates simplest .wasm. Do u think we should use other smith options, like `--bulk-memory-enabled`, `--exceptions-enabled`, `--gc-enabled`, to make generations more complex?